docs/GettingStarted/configuration/ProductionBestPractice.md

```diff
@@ -1,68 +1,52 @@
 
-```
+Additionaly it is recommended to create Persistent Volumes to withstand prometheus container restarts. To enable that the [prometheus_storage](./PelorusCore.md#prometheus_storage) needs to be set to `true` and additionally PVC configuration options controls how it should be created. Relevant options are:
 
-## Configure Long Term Storage (Recommended)
+- [prometheus_storage](./PelorusCore.md#prometheus_storage)
+- [prometheus_storage_pvc_capacity](./PelorusCore.md#prometheus_storage_pvc_capacity)
+- [prometheus_storage_pvc_storageclass](./PelorusCore.md#prometheus_storage_pvc_storageclass)
 
-The Pelorus chart supports deploying a [Thanos](https://thanos.io/) instance for long term storage. It can use any S3 bucket provider. The following is an example of configuring a `pelorus-sample-instance.yaml` file for [NooBaa](https://www.noobaa.io/) with the local s3 service name.
-```yaml
-bucket_access_point: s3.pelorus.svc:443
-bucket_access_key: <your access key>
-bucket_secret_access_key: <your secret access key>
-```
+### Configure Long Term Storage
 
-The default bucket name is thanos. It can be overridden by specifying the following field.
-```yaml
-thanos_bucket_name: <bucket name here>
-```
+The [Configure Prometheus Retention and PV](#configure-prometheus-retention-and-pv) allows to withstand prometheus container restarts, however to ensure data is preserved we recommend deploying Prometheus with the [Thanos](./PelorusCore.md#thanos).
 
-Then deploy Pelorus as described in the [Installation](../../Installation/) doc.
+There are four configuration options to enable [Thanos](./PelorusCore.md#thanos):
 
-If you don't have an object storage provider, we recommend NooBaa as a free, open source option. You can follow our [NooBaa quickstart](../Noobaa) to host an instance on OpenShift and configure Pelorus to use it.
+- [thanos_bucket_name](./PelorusCore.md#thanos_bucket_name)
+- [bucket_access_point](./PelorusCore.md#bucket_access_point)
+- [bucket_access_key](./PelorusCore.md#bucket_secret_access_key)
+- [bucket_secret_access_key](./PelorusCore.md#bucket_secret_access_key)
 
-## Deploying Across Multiple Clusters
+> **Note:** If you don't have an object storage provider, we recommend NooBaa as a free, open source option. You can follow our [NooBaa quickstart](./Noobaa.md) to host an instance on OpenShift and configure Pelorus to use it.
 
-By default, Pelorus will pull in data from the cluster in which it is running, but it also supports collecting data across multiple OpenShift clusters. In order to do this, the thanos sidecar can be configured to read from a shared S3 bucket across clusters. See [Pelorus Multi-Cluster Architecture](../../../Architecture/) for details. You define exporters for the desired metrics in each of the clusters and the main cluster's Grafana dashboard will display a combined view of the metrics collected in the shared S3 bucket via thanos.
+## Example
```

Comment:
one more `#` to put it under core section

Reply:
fixed, pelase re-approve new push.